tests/input/Asset.h

```diff
@@ -11,7 +11,7 @@ struct FFIG_NAME(CDO) CollateralisedDebtObligation : Asset
 {
   CollateralisedDebtObligation() {}
 
-  double PV() const override { return 0.0; }
+  double PV() const override { return 99.99; }
```

Comment:
99.99 seems a better test as it checks that the return value is represented as floating point not integer or pointer.